src/main/resources/templates/error.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Error Page</title>
+</head>
+<body>
+<h1>Oops.. we ran into an error</h1>
+<p>Additional Details about the error</p>
+</body>
+</html>
```

Comment:
[optional] - created an error page that you can redirect the user to in case something goes wrong.